plugin/xcodebuild.vim

```diff
@@ -50,7 +51,17 @@ function! s:assert_project()
 endfunction
 
 function! s:base_command()
-  return 'xcodebuild ' . s:build_target() . ' ' . s:scheme()
+  return 'xcodebuild ' . s:build_dir() . ' ' . s:build_target() . ' ' . s:scheme()
+endfunction
+
+function! s:build_dir()
+  if exists('g:xcodebuild_build_dir')
+    let build_dir = g:xcodebuild_build_dir
+  else
+    let build_dir = s:default_build_dir
```

Comment:
I think I saw this question on a previous PR but the "normal" way to do this would be setting the global var at the top if it didn't exist.